python/GafferSceneTest/SeedsTest.py

```diff
@@ -221,7 +221,7 @@ def testDensityPrimitiveVariable( self ) :
 		# Add the primitive variable, it should take effect.
 
 		primitiveVariables["primitiveVariables"].addChild( Gaffer.NameValuePlug( "d", IECore.FloatData( 0.5 ) ) )
-		self.assertLessEqual( seeds["out"].object( "/plane/seeds" ).numPoints, p.numPoints )
+		self.assertLess( seeds["out"].object( "/plane/seeds" ).numPoints, p.numPoints )
```

Comment:
For this to pass, the `NameValuePlug` constructed on line 223 needs to have the `Dynamic` flag specified. As discussed over chat, we do want to figure out how we can trigger dirty propagation for the addition of non-dynamic plugs, but in this case the plug really should be dynamic anyway (it would fail to serialise without it, and the unit tests should provide reliable API examples for people reading them).